modules/common-fs2/src/main/scala/com/snowplowanalytics/snowplow/enrich/common/fs2/Assets.scala

```diff
@@ -154,7 +154,7 @@ object Assets {
    * Creates an update stream that periodically checks if new versions of assets are available.
    * If that's the case, updates them locally for the enrichments and updates the state.
    */
-  def updateStream[F[_]: ConcurrentEffect: ContextShift: Parallel: Timer](
+  def updateStream[F[_]: ConcurrentEffect: ContextShift: Parallel: Timer: HttpClient](
```

Comment:
As I understand, `HttpClient` is an abstraction of the underlying HTTP client. But in many places we specify directly `Http4sClient` instead of `HttpClient`. I assumed that we do so for code that is used only in enrich fs2 and not in stream enrich. But here this code is used only for enrich fs2 and we use `HttpClient` instead of `Http4sClient`. Is there a reason?